src/domain/wallets/tx-history.ts

```diff
@@ -225,8 +225,18 @@ export const fromLedger = (
   }
 }
 
-const shouldDisplayMemo = (credit: number) => {
-  return credit === 0 || credit >= MEMO_SHARING_SATS_THRESHOLD
+const shouldDisplayMemo = ({
+  memo,
+  credit,
+}: {
+  memo: string | undefined
+  credit: number
+}) => {
+  return (
+    (memo && Object.keys(onboardingEarn).includes(memo)) ||
```

Comment:
instead of `Object.keys(onboardingEarn).includes(memo)` maybe we could have another function `isAuthorizedMemo({memo}): boolean`. it would clarify the intent and also if we want to have other field here (like fees related that could be below threshold) we could do it easily.